storage/innobase/xtrabackup/src/redo_log.cc

```diff
@@ -1289,6 +1305,11 @@ bool Redo_Log_Data_Manager::start() {
 
   debug_sync_point("xtrabackup_pause_after_redo_catchup");
 
+  DBUG_EXECUTE_IF("xtrabackup_reopen_files_after_catchup",
```

Comment:
It looks like the lack of control of debug sync for background threads is making our life difficult to control the threads at specific points. (FYI, you can close the comment after reading :))